dashboard/src/components/ActionOrConsultationName.js

```diff
@@ -1,20 +1,29 @@
-import { Badge } from 'reactstrap';
 import React from 'react';
 import { userState } from '../recoil/auth';
 import { useRecoilValue } from 'recoil';
 import { disableConsultationRow } from '../recoil/consultations';
 
+const getName = (item) => {
+  if (item.name) return item.name;
+  if (!item.isConsultation) return 'Action'; // should never happen
+  return `Consultation ${item.type}`;
```

Comment:
Useless suggestion: unreachable à la fin

```suggestion
  if (item.name) return item.name;
  if (item.isConsultation) return `Consultation ${item.type}`;
  return 'Action'; // should never happen
```